lib/ui/Editor.js

```diff
@@ -76,20 +76,44 @@ Editor.prototype.__proto__ = BaseElement.prototype;
 
 Editor.prototype.open = Promise.method(function (path) {
   var self = this;
+  var initialCursor = {
+    y: 1,
+    x: 1
+  };
+  
   path = util.resolvePath(path);
+  
   return self.ready // ensures textBuf path-changed is triggered in _initHandlers
 
     // a little hacky, but handles errors that text-buffer doesn't
     .then(function () { return fs.openAsync(path, 'r'); })
     .then(function (fd) { return fs.closeAsync(fd); })
     .catch(function (err) {
-      if (err.code === 'ENOENT') return true;
+      var extractPath;
+    
+      if (err.code === 'ENOENT') {
+        extractPath = util.extractCoordsFromPath(path);
+        
+        if (extractPath.extractable) {
+          path = extractPath.cleanPath;
+          
+          initialCursor.y = extractPath.coords[0] || 1;
+          initialCursor.x = extractPath.coords[1] || 1;
+          
+          return false;
+        } else {
+          return true; 
+        }
+      }
       throw err;
     })
-
     .tap(function () { return self.textBuf.setPath(path); })
```

Comment:
`self.textBuf.setPath(util.resolvePath(pathCoords.path));` (should be the only reference to `path` in this function but I could be mistaken)